azure/nat-gateway/output.tf

```diff
@@ -0,0 +1,3 @@
+output "subnet_id" {
+  value = data.azurerm_subnet.natgw.id
```

Comment:
would be better to printout the `ip_address` instead?

Reply:
you're right, I copied the value with the intent of changing it later. I'll update